jwst/outlier_detection/outlier_detection_ifu.py

```diff
@@ -206,8 +208,8 @@ def flag_outliers(self, idet, uq_det, ndet_files,
                 # set all science data that have DO_NOT_USE to NAN
                 sci[bad] = np.nan
 
-                # Compute left and right differences (MIRI dispersion axis = 1)
-                # For NIRSpec dispersion axis = 0, these differences are top, bottom
+                # Compute left and right differences (MIRI dispersion axis = 1 along y axis)
+                # For NIRSpec dispersion axis = 0 (along the x axis), these differences are top, bottom
```

Comment:
```suggestion
                # Compute left and right differences (MIRI dispersion axis = 2, along y axis)
                # For NIRSpec dispersion axis = 1 (along the x axis), these differences are top, bottom
```
The "dispaxis" (or whatever the right name is) keyword values in the header are 1-indexed, not 0-indexed.